PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -1949,7 +1949,7 @@ public void AddShapeButtonClick(Office.IRibbonControl control)
             customShape.AddCustomShape(shapeName, shapeFullName, true);
         }
 
-        private static CustomShapePane InitCustomShapePane()
+        private static CustomShapePane InitCustomShapePane(bool toggleVisible = false)

```

Comment:
This method is mixing the initialization of shapes lab and the UI visibility control. Can u refactor it ?